README.md

```diff
@@ -31,21 +31,25 @@ Having a bug tracking system doesn't make bug handling efficient and CI/CD doesn
 # Tech
 
 - What are the usual stacks used at the company?
-- How do you use source control?
+- Is there default operating system used or is it up to employees to choose one?
+- How do you use source control? Is it git/svn?
 - How do you test code?
 - How do you track bugs?
 - How do you integrate and deploy changes? Is it CI/CD?
+- Are CI/CD pipelines setup under version control / available as code?
 - Is your infrastructure setup under version control / available as code?
 - What's the workflow from the planning to the finished task?
 - How do you prepare for disaster recovery?
 - Is there a standardised development environment? Is it enforced?
 - How quickly can you setup a new local test environment for the product? (minutes / hours / days)
 - How quickly can you respond to security issues in the code or dependencies?
 - Are all developers allowed to have local admin access of their computers?
+- Do you follow ["12-factor app"](https://12factor.net/) methodology?
+- Are developers expected to participate in process of software architecture design?
 
 # The Team
 
-- How is the work organised?
+- How is the work organised? Do you use any issue tracking system like JIRA / Trello?
```

Comment:
Maybe even distinguish between a issue/task management and Product backlog system

Reply:
Good note, gotta change it :)